### ISSUE TITLE
content not displaying properly in browser

### ISSUE THREAD

**Issue by asatvilker** [aws-amplify/amplify-console#967]
**app info**
app id: d1djbarh3iptp7
region: eu-west-2

**problem**
when I deploy my app, it builds successfully and in the verify stage it shows snapshots of the app which look correct (they look as expected). however when I open it up via my url: adamsatvilker.com , it is all over the place, stylesheets are missing? note that I have done many builds before for this same app and it has been successful, it is only now that i have all of a sudden my website is loading with missing css files. 

**error message**
when I checked the console, it says the there is an error in manifest.json. i have looked in the file to see this error and for some reason there is html in the file as apposed to the json content which is meant to be there?

**Comment by hsspain** [comment]
@asatvilker We are looking into this now. Did a re-deploy resolve the issue? Were there any build errors/warning in your logs?

**Comment by asatvilker** [comment]
Hi, 
No unfortunately i tried rebuilding and it made no difference.
The issue started because for a certain route i was getting and access denied error. I tried adding the spa redirect rule stated in the docs and then it gave me access however i got the issue above. I didn’t realise it was the spa rule as when i removed it the styling didnt come back. I eventually realised the browser had cached a recent version of the site so it just kept loading that instead! When I cleared the browser data and then removed the SPA redirect rule it all loads fine again. The rule is definitely causing the issue. I cannot see why or how though.

I did however find an alternative solution. I added my own rule. A 200 rewrite to index.html with the rule: /blog/<*> it now works correctly!
